frontend/public/components/_edit-yaml.scss

```diff
@@ -36,3 +36,22 @@
 .yaml-editor .monaco-editor .margin .current-line {
   background-color: var(--pf-global--BackgroundColor--dark-200);
 }
+
+.yaml-editor__shortcuts {
+  display: inline-block;
+  margin-right: 10px;
```

Comment:
This results in this link not aligning with the left edge of the yaml editor when the divider and second link are not present.
<img width="313" alt="Screen Shot 2019-10-31 at 4 02 52 PM" src="https://user-images.githubusercontent.com/895728/67982260-b80f9180-fbf8-11e9-9d97-8212d6e0dd90.png">
If you put the margin on either side of the divider, you don't need to add it to each link.

Reply:
Good catch, thanks. I wasn't thinking.